apps/docusaurus/docs/nodes/validator-node/operator/delegation-pool-operations.md

```diff
@@ -21,13 +21,22 @@ Before initializing a delegation pool, you need to know the delegation pool addr
 
 - Before you create the delegation pool:
   ```bash
-  aptos account derive-resource-account-address --address <owner_address> --seed "aptos_framework::delegation_pool<SEED>" --seed-encoding utf8
+  aptos account derive-resource-account-address --address <owner_account_address> --seed "aptos_framework::delegation_pool<SEED>" --seed-encoding utf8
   ```
   - The `<SEED>` is a number chosen by you to create the resource account address to host the delegation pool resource. Once you choose a seed, you should use the same value for all following usages.
-- After you create the delegation pool:
-  ```bash
```

Comment:
I don't understand what this command is doing here `aptos account derive-resource-account-address`. If I simply run it I get the following warning:

```
aptos-cli account derive-resource-account-address
error: the following required arguments were not provided:
  --address <ADDRESS>
  --seed <SEED>

Usage: aptos-cli account derive-resource-account-address --address <ADDRESS> --seed <SEED>

For more information, try '--help'.

```

So I'm removing it from the docs

Reply:
Yeah, this looks like a mistake, the above line should be the same as the below one, as both should derive the same.

Reply:
Yeah, this looks out-of-date to me, too 😄